app/components/ViewContainer.js

```diff
@@ -1,19 +1,50 @@
 import React, { Component, PropTypes, View } from 'react-native';
 import AR from './AR.js';
 import Map from './Map.js';
+import DropNewPinButton from '../containers/container_dropNewPin'
 
 export default class ViewContainer extends Component {
-  constructor() {
-    super();
+  constructor(props) {
+    super(props);
     this.state = {
-      view: 'map'
+      view: 'map',
+      currLoc: null,
     };
+    this.fireRef = new Firebase("https://interruptedlobster.firebaseio.com/");
   }
+
+  componentDidMount() {
+    navigator.geolocation.getCurrentPosition(
+      (position) => {
+        var coords = {};
+        coords.longitude = position.coords.longitude;
+        coords.latitude = position.coords.latitude;
+        this.setState({currLoc: coords});
+      },
+      (error) => {
+        alert(error.message)
+      },
+      {enableHighAccuracy: true, timeout: 20000, maximumAge: 1000}
+    );
+
+    this.watchID = navigator.geolocation.watchPosition(
+      (position) => {
+        var coords = {};
+        coords.longitude = position.coords.longitude;
+        coords.latitude = position.coords.latitude;
+        this.setState({currLoc: coords});
+        // update firebase with current position
+        // this.currentPosition.set(position);
+      }
+    );
+  }
```

Comment:
All of this stuff is in componentDidMount() -- will it actually update our position as we walk around, or will it only set our position to the point in time when the component was mounted to the DOM?